pom.xml

```diff
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.springframework.boot</groupId>
     <artifactId>spring-boot-starter-parent</artifactId>
-    <version>1.5.9.RELEASE</version>
+    <version>2.0.6.RELEASE</version>
```

Comment:
I don't think this will be backward compatible with spring-boot 1.x
would suggest forking this and create common-plugin-2